Description: Remove all references to non-dfsg files
 Clementine use non-dfsg files 'hypnotoad.gif' and 'lumberjacksong.txt' as 
 "Easter eggs" in clementine interface.
Author: Thomas PIERSON <web.pierson@gmail.com>
Last-Update: 2011-05-24
--- a/data/data.qrc
+++ b/data/data.qrc
@@ -261,7 +261,6 @@
         <file>icons/22x22/go-down.png</file>
         <file>icons/32x32/go-down.png</file>
         <file>icons/48x48/go-down.png</file>
-        <file>hypnotoad.gif</file>
         <file>blank.ttf</file>
         <file>schema/schema-16.sql</file>
         <file>icons/22x22/media-eject.png</file>
@@ -282,7 +281,6 @@
         <file>providers/mtvmusic.png</file>
         <file>providers/cdbaby.png</file>
         <file>providers/echonest.png</file>
-        <file>lumberjacksong.txt</file>
         <file>schema/schema-18.sql</file>
         <file>star-off.png</file>
         <file>star-on.png</file>
--- a/src/ui/mainwindow.ui
+++ b/src/ui/mainwindow.ui
@@ -473,8 +473,6 @@
      <string>&amp;Extras</string>
     </property>
     <addaction name="action_rain"/>
-    <addaction name="action_hypnotoad"/>
-    <addaction name="action_kittens"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menu_tools">
--- a/src/widgets/fullscreenhypnotoad.cpp
+++ b/src/widgets/fullscreenhypnotoad.cpp
@@ -20,7 +20,7 @@
 #include <QMovie>
 
 FullscreenHypnotoad::FullscreenHypnotoad()
-  : movie_(new QMovie(":/hypnotoad.gif"))
+  : movie_(new QMovie(":/spinner.gif"))
 {
   setMovie(movie_);
   setScaledContents(true);
--- a/src/widgets/nowplayingwidget.cpp
+++ b/src/widgets/nowplayingwidget.cpp
@@ -36,7 +36,7 @@
 
 const char* NowPlayingWidget::kSettingsGroup = "NowPlayingWidget";
 
-const char* NowPlayingWidget::kHypnotoadPath = ":/hypnotoad.gif";
+const char* NowPlayingWidget::kHypnotoadPath = ":/spinner.gif";
 
 // Space between the cover and the details in small mode
 const int NowPlayingWidget::kPadding = 2;
--- a/src/songinfo/songinfosettingspage.cpp
+++ b/src/songinfo/songinfosettingspage.cpp
@@ -42,9 +42,6 @@
   connect(ui_->providers, SIGNAL(itemChanged(QListWidgetItem*)),
           SLOT(ItemChanged(QListWidgetItem*)));
 
-  QFile song_info_preview(":/lumberjacksong.txt");
-  song_info_preview.open(QIODevice::ReadOnly);
-  ui_->song_info_font_preview->setText(QString::fromUtf8(song_info_preview.readAll()));
 
   connect(ui_->song_info_font_size, SIGNAL(valueChanged(double)), SLOT(FontSizeChanged(double)));
 }
